### PR TITLE
Updating README with upstream docs build instructions (noref)

### PR DESCRIPTION
**Pull request by asettle**
Updating the README with instructions on how to build upstream guides for future ref. I would recommend testing these instructions before merge, please.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -16,11 +16,20 @@ The following guides are available:
 * SUSE OpenStack Cloud, General:
 ** Security Guide
 ** Supplement to Administrator Guide and User Guide
+
+These guides are all maintained and created in this GitHub
+repository .
+
+The folowing guides are available upstream:
```
Comment (sknorr) [inline README.adoc]: * typo: fol**l**owing
* Maybe reword to "The following guides are built directly from upstream OpenStack sources:" to clarify what "available" means and what upstream you are talking about

File: README.adoc [hunk 2]
```diff
@@ -16,11 +16,20 @@ The following guides are available:
 * SUSE OpenStack Cloud, General:
 ** Security Guide
 ** Supplement to Administrator Guide and User Guide
+
+These guides are all maintained and created in this GitHub
+repository .
```
Comment (sknorr) [inline README.adoc]: No space before the period, please.

File: README.adoc [hunk 3]
```diff
@@ -16,11 +16,20 @@ The following guides are available:
...
 ** Administration Guide
 ** User Guide
 
-Released versions of these guides have been published at
-https://www.suse.com/documentation/.
+These guides are all maintained upstream in the various OpenStack
+project repositories. Released versions of these guides have been
+generated using the `suse_sphinx_theme` and published at
+https://www.suse.com/documentation/. This setup can be found
+here: https://github.com/SUSE-Cloud/doc-cloud-upstream
```
Comment (sknorr) [inline README.adoc]: This is a bit unclear, maybe make two paragraphs out of it, as:

* only the upstream stuff is built with the SUSE Sphinx theme
* both SUSE and upstream guides are published at the same location

File: README.adoc [hunk 4]
```diff
@@ -115,6 +124,25 @@ our daps2docker project: https://github.com/openSUSE/daps2docker
 2. Clone the daps2docker repository.
 3. Run  `./daps2docker.sh /PATH/TO/DOC-DIR` or `/daps2docker.sh /PATH/TO/DC-FILE`.
 
+Building upstream docs
```
Comment (sknorr) [inline README.adoc]: Given that these upstream guides are in another repo, I'd suggest not to document the procedure in this repo but to add a repo readme in the other repo.
Otherwise, the content of this readme will just become outdated when a developer changes anything and nobody from the developer team will notice. If you put these instructions in the other repo's readme, 

This repo readme should have a link to the other readme though.

File: README.adoc [hunk 5]
```diff
@@ -115,6 +124,25 @@ our daps2docker project: https://github.com/openSUSE/daps2docker
...
+~~~~~~~~~~~~~~~~~~~~~~
+
+If you're required to build the upstream Administration and User Guides, you can build and view each individual guide upstream: https://docs.openstack.org/doc-contrib-guide/docs-builds.html 
+
+If you want to build the SUSE version that is published at suse.com/documentation, perform the following:
```
Comment (sknorr) [inline README.adoc]: Maybe reword to "...the SUSE version, equivalent to what is published at ..."

File: README.adoc [hunk 6]
```diff
@@ -115,6 +124,25 @@ our daps2docker project: https://github.com/openSUSE/daps2docker
...
+
+1. Clone https://github.com/SUSE-Cloud/doc-cloud-upstream
```
Comment (sknorr) [inline README.adoc]: Do I need to install any dependencies?

File: README.adoc (hunk 4 above)
Comment (asettle) [reply README.adoc]: So, I agree and disagree at the same time. There is a README in the other repo (which, as we discussed, needs to be moved to the top-level) but I think it's a good thing these instructions are here. It's a consistent reference point to *all* documentation for cloud. A lot of people will look here for these instructions. The other repo is just a body, mostly?

**Comment by asettle** [comment]
@sknorr https://github.com/SUSE-Cloud/doc-cloud-upstream/pull/3

**Comment by asettle** [comment]
Hm @sknorr travis keeps timing out: https://travis-ci.org/SUSE-Cloud/doc-cloud/jobs/569167769

**Comment by csymons-suse** [comment]
On 8/8/19 5:32 AM, Stefan Knorr wrote:
> *@sknorr* approved this pull request.
> 
> Me gusta.
> 
> Point about this being the being the central doc repo is taken. However,
> still: between here, the doc-cloud-upstream repo and Confluence (which
> csymons asked for), you are now putting the instructions in three
> places. Updating all of those always is going to be tedious.

Good point.

okay with me to blow off Confluence for this. The information is really
only interesting to the doc team, specifically the cloudists. Maybe just
complete information in doc-cloud-upstream with a pointer to that repo
in the main doc-cloud repo.

**Comment by asettle** [comment]
Fair points taken, team! I'll switcheroo :)

File: README.adoc [hunk 7]
```diff
@@ -131,19 +131,7 @@ Building upstream docs
...
-   
-   $ cd scripts
-   
-3. Run `./gen_docs.sh`
-
-   This process clones every repository required, and builds them using the `suse_sphinx_theme`. You need to allocate at
-   least an hour for the process to complete.
-   
-4. You can find the finished product in the `/scripts/build/` folder.
+If you want to build the SUSE version, equivalent to what is published at at suse.com/documentation, see the instructions on how to build here: https://github.com/SUSE-Cloud/doc-cloud-upstream/README
```
Comment (sknorr) [inline README.adoc]: This link does not work. :(
There is a README file in the `rocky` branch but not in `master`.
(Also, earlier, I failed to grasp that the README was in a different branch and not in a different directory :woman_facepalming: .)

Comment (asettle) [reply README.adoc]: Ah yes thank you, good catch. The branch needs to stay as is. It's the main branch of the repo, not master.
